core/src/main/java/org/elasticsearch/index/mapper/DocumentMapper.java

```diff
@@ -56,7 +56,7 @@
 
         private final RootObjectMapper rootObjectMapper;
 
-        private Map<String, Object> meta = emptyMap();
+        private Map<String, Object> meta;
```

Comment:
I think this initialization should stay, for the case meta is not defined in the json?

Reply:
@rjernst I was thinking of using `null` for this case. Or is empty map better?
And which value must be used for unset meta?

Reply:
I'm fine with null. I had thought changing this from emtpy map to null would require updating Mapping.toXContent. However, it looks like it currently handles both cases. I would update that to remove the empty case (which means explicitly putting an empty _meta would be serialized, not disappear).